vocode/streaming/streaming_conversation.py

```diff
@@ -993,6 +994,9 @@ async def start(self, mark_ready: Optional[Callable[[], Awaitable[None]]] = None
         self.transcriptions_worker.block_inputs = False
 
     async def send_initial_message(self, initial_message: BaseMessage):
+        initial_message_span = start_span_in_ctx(
+            name="send_initial_message", parent_span=self.conversation_span
+        )
```

Comment:
Starts a span in context. Should *not* throw if tracing is not configured correctly. see `start_span_in_ctx` defined in utils for more info.